spec.emu

```diff
@@ -0,0 +1,167 @@
+  <h1>Properties of the ArrayBuffer Prototype Object</h1>
+  <emu-clause id="sec-properties-of-the-arraybuffer-prototype-@@slice">
+    <h1>ArrayBuffer.prototype [ @@slice ] (start, end)</h1>
+    <p> The initial value of the @@slice property is the same function object as the initial value of the `ArrayBuffer.prototype.slice` property. </p>
+  </emu-clause>
+</emu-clause>
+
+<emu-clause id="sec-properties-of-the-sharedarraybuffer-prototype-object">
+  <h1>Properties of the SharedArrayBuffer Prototype Object</h1>
+  <emu-clause id="sec-properties-of-the-sharedarraybuffer-prototype-@@slice">
+    <h1>SharedArrayBuffer.prototype [ @@slice ] (start, end)</h1>
+    <p> The initial value of the @@slice property is the same function object as the initial value of the `SharedArrayBuffer.prototype.slice` property. </p>
+  </emu-clause>
+</emu-clause>
+
+<emu-clause id="sec-left-hand-side-expressions">
+  <h1>Left-Hand-Side Expressions</h1>
+  <h2>Syntax</h2>
+  <emu-grammar>
+    MemberExpression[Yield, Await] :
+      PrimaryExpression[?Yield, ?Await]
+      MemberExpression[?Yield, ?Await] `[` Expression[+In, ?Yield, ?Await] `]`
+      MemberExpression[?Yield, ?Await] `.` IdentifierName
+      MemberExpression[?Yield, ?Await] TemplateLiteral[?Yield, ?Await]
+      SuperProperty[?Yield, ?Await]
+      MetaProperty
+      `new` MemberExpression[?Yield, ?Await] Arguments[?Yield, ?Await]
+      <ins>MemberExpression[?Yield, ?Await] `[` Expression[+In, ?Yield, ?Await]? `:` Expression[+In, ?Yield, ?Await]? `]` </ins>
+  </emu-grammar>
+
+  <emu-grammar>
+    CallExpression[Yield, Await] :
+      CoverCallExpressionAndAsyncArrowHead[?Yield, ?Await] #callcover
+      SuperCall[?Yield, ?Await]
+      ImportCall[?Yield, ?Await]
+      CallExpression[?Yield, ?Await] Arguments[?Yield, ?Await]
+      CallExpression[?Yield, ?Await] `[` Expression[+In, ?Yield, ?Await] `]`
+      CallExpression[?Yield, ?Await] `.` IdentifierName
+      CallExpression[?Yield, ?Await] TemplateLiteral[?Yield, ?Await, +Tagged]
+      <ins>CallExpression[?Yield, ?Await] `[` Expression[+In, ?Yield, ?Await]? `:` Expression[+In, ?Yield, ?Await]? `]`</ins>
+  </emu-grammar>
+
+  <emu-clause oldids="sec-static-semantics-static-semantics-isvalidsimpleassignmenttarget" id="sec-static-semantics-static-semantics-assignmenttargettype">
+    <h1>Static Semantics: AssignmentTargetType</h1>
+    <emu-see-also-para op="AssignmentTargetType"></emu-see-also-para>
+    <emu-grammar>
+      MemberExpression :
+        MemberExpression `[` Expression? `:` Expression? `]`
+
+      CallExpression :
+        CallExpression `[` Expression? `:` Expression? `]`
+    </emu-grammar>
+    <emu-alg>
+      1. Return ~invalid~.
+    </emu-alg>
+  </emu-clause>
+
+  <emu-clause id="sec-slice-notation-runtime-semantics-evaluation">
+    <h1>Runtime Semantics: Evaluation</h1>
+    <emu-grammar> MemberExpression : MemberExpression `[` Expression? `:` Expression? `]`</emu-grammar>
```

Comment:
```suggestion
    <emu-grammar> MemberExpression : MemberExpression `[` AssignmentExpression? `:` AssignmentExpression? `]`</emu-grammar>
```

Reply:
is the space character in `<emu-grammar> MemberExpression` necessary?

Reply:
It shouldn't be, AFAIK.